assembly/src/testing.rs

```diff
@@ -27,7 +27,7 @@ use crate::diagnostics::reporting::set_panic_hook;
 ///
 /// Some of the assertion macros defined above require a [TestContext], so be aware of that.
 pub struct TestContext {
-    source_manager: Arc<dyn SourceManager + Send + Sync>,
+    source_manager: Arc<dyn SourceManagerSync>,
```

Comment:
Same question here with regard to whether the test context really needs `SourceManagerSync`.

Reply:
Fixed